<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BkgroupWindow</class>
 <widget class="QDialog" name="BkgroupWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widgetTop" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>1006</width>
     <height>45</height>
    </rect>
   </property>
   <widget class="QPushButton" name="btnPeople">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labTitle">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>761</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReTurnMenu">
    <property name="geometry">
     <rect>
      <x>923</x>
      <y>0</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <zorder>labTitle</zorder>
   <zorder>btnPeople</zorder>
   <zorder>btnReTurnMenu</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
